### PR TITLE
Add analytics events for cloud onboarding

### PR DESCRIPTION
**Pull request by schustmi**
## Describe changes
I implemented/fixed _ to achieve _.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have based my new branch on `develop` and the open PR is targeting `develop`. If your branch wasn't based on develop read [Contribution guide on rebasing branch to develop](https://github.com/zenml-io/zenml/blob/main/CONTRIBUTING.md#-pull-requests-rebase-your-branch-on-develop).
- [ ] If my changes require changes to the dashboard, these changes are communicated/requested.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: src/zenml/zen_server/auth.py [hunk 1]
```diff
@@ -609,9 +609,12 @@ def authenticate_external_user(external_access_token: str) -> AuthContext:
...
             context.user_id = user.id
             context.identify(
-                traits={"email": user.email, "source": "external_auth"}
+                traits={
+                    "email": external_user.email,
+                    "source": "external_auth",
+                }
             )
-            context.alias(user_id=user.id, previous_id=external_user.id)
+            context.alias(user_id=external_user.id, previous_id=user.id)
```
Comment (bcdurak) [inline src/zenml/zen_server/auth.py:617]: If I recall correctly, changing the order of these two messed up the identification process before when we first implemented the first version. Why do we have to change this now?

**Review by bcdurak — COMMENTED** [review]
Just had a question regarding alias. Otherwise, everything looks good.

File: src/zenml/zen_server/auth.py (hunk 1 above)
Comment (schustmi) [reply src/zenml/zen_server/auth.py:617]: I think this is actually the wrong way round and messed with ortto as far as I can see.

**Review by AlexejPenner — APPROVED** [review]
:seal:

**Pull request merged**